common/core/desktop/meson.build

```diff
@@ -6,10 +6,10 @@
 #
 
 project('keyboardprocessor', 'cpp', 'c',
-        # Get version number from file (VERSION.md is in this directory for source package)
+        # Get version number from file (VERSION.md is in root of repo)
         # Fallback to "more" for Windows compatbility
         version: run_command(find_program('cat', 'more'),
-                 files('VERSION.md')).stdout().strip(),
+                 files('../../../VERSION.md')).stdout().strip(),
```

Comment:
I think @ermshiperete intentionally used `'VERSION.md'` in #2843 to fix the failing Linux packaging builds

Reply:
Yes, I did. This fixes the Linux package build, but unfortunately breaks things everywhere else unless you (manually) copy VERSION.md... I'm working on a better solution that should work everywhere...

Reply:
I take it meson will choose the correct script in `find_program` based on the OS? Because the documentation is a little vague on that point.

Reply:
It takes the first it finds and can execute. On Windows (at least on my machine) it finds both .bat and .sh, but has problems executing .sh (runs without obvious error, but doesn't seem to DO anything) which is why I put .bat first. On Linux it ignores the .bat file.